[CN-448] Update WAN replication page and sample yaml (#54)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/wan-replication.adoc

= Configuring WAN Replication

> :description: pass:q[WAN Replication allows you to keep Hazelcast clusters in sync across multiple sites or cloud deployments. In Hazelcast Platform Operator, you can create a `WanReplication` resource to replicate multiple maps from multiple source clusters to a single target Hazelcast cluster. ]

Reviewer: ```suggestion
:description: pass:q[WAN replication allows you to keep Hazelcast clusters in sync across multiple sites or cloud deployments. In Hazelcast Platform Operator, you can create a `WanReplication` resource to replicate multiple maps from multiple source clusters to a single target Hazelcast cluster. ]
```

|===
|Field|Description

|`resources`

> |The list of the custom `Map` or `Hazelcast` resource to which WAN replication applies.

Reviewer: ```suggestion
|The list of custom `Map` or `Hazelcast` resources to which WAN replication applies.
```

- `name`: The name of custom resource to which WAN replication applies.

> - `kind`: The kind of custom resource. Either `Map`(default) or `Hazelcast`. The kind `Hazelcast` leads to apply WAN replication to the all custom maps created under the given Hazelcast resource.

Reviewer: ```suggestion
- `kind`: The type of custom resource. Either `Map`(default) or `Hazelcast`. Use the  `Hazelcast` kind to apply WAN replication to all custom maps created under the given `Hazelcast` resource.
```

[source,shell]
----

> kubectl get wanreplication wanreplication-sample

Reviewer 2: ```suggestion
kubectl get wanreplication wanreplication-sample -o wide
```

Or we can remove the `message` column from the output of this command.
